client/my-sites/subscribers/mutations/use-subscriber-remove-mutation.ts

```diff
@@ -15,21 +14,14 @@ type ApiResponseError = {
 	message: string;
 };
 
-const useNewHelper = config.isEnabled( 'subscribers-helper-library' );
-
 const getEmailSubscriptionId = ( subscriber: Subscriber ): number => {
-	if ( useNewHelper ) {
-		// For new helper library, use email_subscription_id if it exists, otherwise use subscription_id
-		return subscriber.email_subscription_id || subscriber.subscription_id || 0;
-	}
-	return subscriber.subscription_id || 0;
+	// `subscription_id` is from the old API endpoint response.
```

Comment:
Is it better to add JSDoc comments in `Subscriber` type? It will give use this info throughout the app where this type is used.